Client/Extensions/UITableViewCell+Extension.swift

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension UITableViewCell {
+    private static var customTopSeparatorKey = "customTopSeparatorKey"
+    private static var customBottomSeparatorKey = "customBottomSeparatorKey"
+
+    var customTopSeparatorView: UIView? {
+        get { return objc_getAssociatedObject(self, &UITableViewCell.customTopSeparatorKey) as? UIView }
+        set { objc_setAssociatedObject(self, &UITableViewCell.customTopSeparatorKey, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC) }
```

Comment:
Could we avoid `objc_setAssociatedObject` usage? If this code isn't in an extension, but rather in the cell class `TwoLineImageOverlayCell` then this would make it easier. If we need the code in other cells in the future, then we'll adjust when the time comes.